ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue

```diff
@@ -139,7 +139,7 @@
               ref="submittingPartyAddress"
               id="submitting-party-address"
               :schema="PartyAddressSchema"
-              :value="submittingParty.address"
+              :value="getMhrRegistrationSubmittingParty.address"
```

Comment:
Why not keep the value as it was `submittingParty`, as it already exists in the `localState` as `getMhrRegistrationSubmittingParty`?